x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -803,7 +803,7 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
                 </EuiFlexItem>
               </EuiFlexGroup>
               <EuiSpacer />
-              <TabNavigationWithBreadcrumbs navTabs={pageTabs} />
+              <TabNavigationWithBreadcrumbs navTabs={pageTabs} isExistingRule={isExistingRule} />
```

Comment:
@gitstart This is a generic component, we should not use it to carry rules-specific properties. As @banderror pointed out, common breadcrumbs implementation must also stay generic.
 
The proper way to do this is by adding `isExistingRule` in the _SpyRoute_ `state` at line 897:
```
<SpyRoute pageName={SecurityPageName.rules} state={{ ruleName: rule?.name, isExistingRule }} />
``` 

Then implement the rules-specific logic inside the rule's trailing breadcrumbs function at:
 
https://github.com/elastic/kibana/blob/f1b0dd4720334038663a57bc6458f3e000811ee1/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/utils.ts#L46-L49

We'll have the `params.state.isExistingRule` available there. We need to update the `RouteSpyState` accordingly.